enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/state/ClusteringModule.java

```diff
@@ -62,6 +64,10 @@ public ClusteringModule( DiscoveryServiceFactory discoveryServiceFactory, Member
         FileSystemAbstraction fileSystem = platformModule.fileSystem;
         HostnameResolver hostnameResolver = chooseResolver( config, logProvider, userLogProvider );
 
+        String dbName = config.get( CausalClusteringSettings.database );
+        SimpleStorage<DatabaseName> dbNameStorage =
+                new SimpleFileStorage<>( fileSystem, clusterStateDirectory, DB_NAME, new DatabaseName.Marshal(), logProvider );
+
```

Comment:
This also would be clearer if it was immediately before `new ClusterBinder` where it is used.

Reply:
👌